css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  flex-basis: 25%;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.padding {
+  padding-right: 2.5rem;
+}
+
+img {
+  width: 100%;
+}
+
+span {
+  color: #ddd;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align {
+  text-align: left;
+  align-items: flex-end;
+  align-self: flex-end;
+}
+
+.a,
+.b,
+.c,
+.d,
+.e {
+  display: inline;
+  color: black;
+}
```

Comment:
These class names are not descriptive at all. Could you make this one meaningfully-named class that you could apply to all these elements?

Reply:
<img width="489" alt="Screen Shot 2022-07-15 at 10 37 03 AM" src="https://user-images.githubusercontent.com/97363006/179279116-69d099ab-6663-4b2f-8e4d-697252b169c6.png">